binder/views.py

```diff
@@ -1202,6 +1211,12 @@ def _multi_put_save_objects(self, ordered_objects, objects, request):
 				validation_errors.append(e)
 			if oid < 0:
 				new_id_map[(model, oid)] = obj.id
+				for base in getmro(model)[1:]:
```

Comment:
You use getmro here, while you have defined a custom `getsubclasses` method previously. Is there a specific difference between the two methods, or are they the same?

From what I understand they basically do the same, in which case it would be better to be consistent.

Reply:
They actually do the reverse, `getmro(A)` will give all classes that `A` inherits from while `getsubclasses(A)` will give all classes that inherit from `A`.